### ISSUE TITLE
[Docs] Broken Links

### ISSUE THREAD

**Issue by cooki35** [qtile/qtile#4384]
# TLDR:

- [ ] Fix http://docs.qtile.org/en/latest/manual/commands/scripting.html in https://github.com/qtile/qtile/blob/6357381aa54b47f9e5ecb0f434d728dd165a83fb/libqtile/extension/base.py#L95
- [ ] Add quotes to https://github.com/qtile/qtile/blob/6357381aa54b47f9e5ecb0f434d728dd165a83fb/docs/manual/config/lazy.rst?plain=1#L8

# The Issues:
While looking at the documentation I discovered two broken links.

## RunCommand
There is a broken (404) link in the documentation of the RunCommand extension (Configuration -> Built-in Extensions -> RunCommand).
https://docs.qtile.org/en/latest/manual/ref/extensions.html#libqtile.extension.CommandSet
The documentation states the following:
`Also consider simply using lazy.spawn() or writing a [client](http://docs.qtile.org/en/latest/manual/commands/scripting.html).`
But http://docs.qtile.org/en/latest/manual/commands/scripting.html does not exist.
We should fix this in: https://github.com/qtile/qtile/blob/6357381aa54b47f9e5ecb0f434d728dd165a83fb/libqtile/extension/base.py#L95
Should we refer to https://docs.qtile.org/en/latest/manual/commands/advanced.html#tying-it-together-command-client and use `:doc:`?

## Lazy objects
There is no link when referencing the Commands API part of the documentation in https://docs.qtile.org/en/latest/manual/config/lazy.html. Should be fixed in: https://github.com/qtile/qtile/blob/6357381aa54b47f9e5ecb0f434d728dd165a83fb/docs/manual/config/lazy.rst?plain=1#L8
Should we change it to: ```:doc:`commands API </manual/commands/api/index>` ```?

# Versions and Logs:
qtile version: not relevant
logs: not relevant
branch: master
Last time I checked the documentation: 2023-08-07 10:30 UTC

### Required:

- [X] I have searched past issues to see if this bug has already been reported.

**Comment by elParaguayo** [comment]
Thanks. Do you want to submit a pull request with the required changes?
